src/modules/image-upload/image-upload.js

```diff
@@ -98,3 +103,40 @@ if (!config.s3) {
 	}, APP_PRIORITIES.IMAGE_UPLOAD);
 	winston.info('Image upload is ready');
 }
+
+var isS3Url = (url) => {
+	if (/https?\:\/\/.*\.amazonaws\.com\//.test(url)) {
+		return true;
+	}
+	return false;
+};
+
+
+bus.on('postchange', async ({ entities }) => {
```

Comment:
this should be disabled if no s3 config is present. Move inside the `if (!config.s3)` condition above.